views/secretWord.ejs

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi"
+      crossorigin="anonymous"
+    />
+    <title>Jobs List</title>
+  </head>
+  <body>
+    <p>The secret word is: <%- include("partials/head.ejs") %> 
+        <%- include("partials/header.ejs") %> 
+        <p>The secret word is: <%= secretWord %></p> 
+        <p>Would you like to change it?</p> 
+        <form method="POST"> 
+            <label name="secretWord"></label> 
+            <input name="secretWord" /><br /> 
+            <button>Submit</button> 
+        </form> <%- include("partials/footer.ejs") %></p>
+    <p>Would you like to change it?</p>
+    <form method="POST">
```

Comment:
This is not correct.  In ejs, it is important to specify a value for the action attribute of the form.  The value is the route that is to be invoked, when the submit button is pressed.  So you might have action="/changeWord", and then in your express code you handle the post request for the "/changeWord" route.